toolchain/check/cpp/import.cpp

```diff
@@ -2014,18 +2016,10 @@ auto ImportCppFunctionDecl(Context& context, SemIR::LocId loc_id,
       SemIR::ClangDeclKey::ForFunctionDecl(clang_decl, num_params));
 }
 
-// Maps `clang::AccessSpecifier` to `SemIR::AccessKind`.
-static auto MapAccess(clang::AccessSpecifier access_specifier)
+// Deduces the access given UnresolvedSetIterator.
+static auto MapAccess(clang::UnresolvedSetIterator iterator)
     -> SemIR::AccessKind {
-  switch (access_specifier) {
-    case clang::AS_public:
-    case clang::AS_none:
-      return SemIR::AccessKind::Public;
-    case clang::AS_protected:
-      return SemIR::AccessKind::Protected;
-    case clang::AS_private:
-      return SemIR::AccessKind::Private;
-  }
+  return DeduceClangAccess(iterator.getAccess(), iterator->getAccess());
```

Comment:
The difference between `.getAccess` and `->getAccess` seems subtle, perhaps document what's occurring here where you do it?